Makefile.am

```diff
@@ -338,7 +338,7 @@ clean-docs:
 clean-local: clean-docs
 	rm -rf coverage_percent.txt test_bitcoin.coverage/ total.coverage/ fuzz.coverage/ test/tmp/ cache/ $(OSX_APP)
 	rm -rf test/functional/__pycache__ test/functional/test_framework/__pycache__ test/cache share/rpcauth/__pycache__
-	rm -rf osx_volname dist/
+	rm -rf osx_volname dist/ test/lint/test_runner/target/
```

Comment:
We also don't delete the `test/lint/__pycache__`, maybe a good opportunity to add it?

Reply:
yepp, thanks, added!